src/components/targets/CreateTarget.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+import '../users/user-form.scss';
+import './create-target.scss';
+import backImage from '../../assets/back.png';
+import targetImage from '../../assets/newTarget.png';
+import CreateTargetForm from './CreateTargetForm';
+
+const CreateTarget = ({ newTargetlatlng, setNewTargetlatlng }) => {
+  const intl = useIntl();
+
+  const handleGoBack = () => {
+    setNewTargetlatlng(null);
+  };
+
+  return (
+    <div className="create-target">
+      <div className="create-target__title-bar">
+        <div className="create-target__title-bar-back">
+          <input
```

Comment:
Is there a special reason why the go back button is an input tag?

Reply:
Because I couldn't find the way to put an image as a button. Is there a better way?